src/main/java/org/polystat/Polystat.java

```diff
@@ -91,9 +99,23 @@ public void exec(final String... args) throws Exception {
                 Paths.get(args[0]), Paths.get(args[1])
             );
             for (final Analysis analysis : Polystat.ALL) {
-                final List<String> errors = new ListOf<>(
-                    analysis.errors(xmir, "\\Phi.foo")
-                );
+                List<String> errors = new ListOf<>();
+
+                try {
+                    errors = new ListOf<>(
+                        analysis.errors(xmir, "\\Phi.foo")
+                    );
+                }
+                catch (Exception e) {
+                    Logger.error(
+                        this,
+                        "Analysis method \"%s\" finished with exception:\n%s",
+                        analysis.name(),
+                        e.toString()
+                    );
+                    continue;
```

Comment:
@Sitiritis why such a swallow of exceptions here? I'd rather let the application die at this point

Reply:
Since this is the part of the code responsible for running the analyzers and printing the results I would like to explicitly handle the case that some analyzer have failed and print a nice-looking message instead of a stack trace.

Reply:
If I catch `Exception`, check style complains that I can't do that. Maybe this particular check should be turned off? Because disallowing to handle `Exception` does not make sense for me in this context, since it is possible to get any subclass of `Exception`

Reply:
@Sitiritis don't catch anything. See: https://www.yegor256.com/2015/07/09/catch-if-cant-otherwise.html